packages/playground/src/layouts/DefaultHeaderLayout.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react'
+
+interface DefaultHeaderProps {
+  children: ReactNode
+}
+
+// This is empty container with padding without header
+const DefaultHeaderLayout: React.FC<DefaultHeaderProps> = ({ children }) => {
+  return <div className="p-5 h-16">{children}</div>
```

Comment:
"layout" components should typically render one fixed component, and then render children or `<Outlet />` for the body